renovate-presets/default.json5

```diff
@@ -128,6 +129,7 @@
       "enabled": false,
       "matchPackageNames": [
         "@types/node",
+        "bazel",
```

Comment:
I am assuming this is fine to update minor and patch

Reply:
Yeah, also we won't have many `@bazel` deps in the future anymore